easybuild/easyconfigs/t/TAMkin/TAMkin-1.0.8-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PythonPackage"
+
+name = 'TAMkin'
+version = '1.0.8'
+
+homepage = 'http://molmod.github.io/tamkin/'
+description = """TAMkin is a post-processing toolkit for normal mode analysis,
+ thermochemistry and reaction kinetics. It uses a Hessian computation from a
+ standard computational chemistry program as its input."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/molmod/tamkin/releases/download/v%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+python = "Python"
+pythonversion = '2.7.10'
+pythonshortversion = ".".join(pythonversion.split(".")[:-1])
```

Comment:
yes, you can do:
```python
dependencies = [
    ('Python', '2.7.10')
]
versionsuffix = "-Python-%(pyver)s"
```
As a complete example: https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/m/matplotlib/matplotlib-1.5.1-intel-2016a-Python-2.7.11.eb